templates/Avent/2019/day_08.html.twig

```diff
@@ -2,7 +2,7 @@
 
 {% set year = 2019 %}
 
-{% block article_title "REX : de Symfony 2 à Symfony 4 et au-delà<br/>(We are back !)" %}
+{% block article_title "REX : de Symfony 2 à Symfony 4 et au-delà (We are back !)" %}
```

Comment:
in the article itself, the title was one one line (see line 8), so I considered this the simplest way to fix it.